### ISSUE TITLE
Arch linux and OnePlus One issues

### ISSUE THREAD

**Issue by ntzrmtthihu777** [M0Rf30/android-udev-rules#70]
Using this file as provided by the android-udev r179-1 package does not change either /dev/android_adb or /dev/libmtp-%k symlink's group ownership to adbusers

**Comment by JoesCat** [comment]
Some distros don't have an adbusers group and therefore you will need to create it.
See if such a group exists in your distro first, if not, you will need to create adbusers group.

If you added one, or already have one, you will see the symlink (shortcut) running as root.root
'''c
$ ls -l /dev/
lrwxrwxrwx  1 root root            15 Jan 18 21:41 android_adb -> bus/usb/005/004
'''
but the actual connection will be running as root.adbusers
'''c
$ ls -l /dev/bus/usb/005/
crw-rw---- 1 root adbusers 189, 515 Jan 18 21:41 004
'''

**Comment by JoesCat** [comment]
...never mind...
...seems the code changed substantially since I last saw it, and could use a large parsing of repeated code....looks like a large patch can be thrown in.... meantime, for your code, add:
ENV{adb_user}="yes"
to the end of the line where it matches your oneplus and see if that cures it for you.

**Comment by ntzrmtthihu777** [comment]
Actually no, this was my fault here. Yes, `/dev/android_adb` was `root:root`, but the `/dev/bus/XXX/YYY` node was owned correctly, which was confusing me. The actual issue turned out to be how I was starting my DE; using startx/.xinitrc and `exec i3`, it turns out I had to `exec dbus-launch i3` in order to get libmtp to play nice with thunar and some other bs, and as far as I could figure (incorrectly) the issue was arising from some odd interaction between the default libmtp udev rules and yours here.

In any case, all is well, though I've found another issue I'll make a new issue for.
